dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java

```diff
@@ -186,7 +186,7 @@ public void doSaveProperties(long version) {
             } else {
                 registryCacheExecutor.execute(new SaveProperties(lastCacheChanged.incrementAndGet()));
             }
-            logger.warn("Failed to save registry store file, cause: " + e.getMessage(), e);
+            logger.warn("Failed to save registry cache file, cause: " + e.getMessage(), e);

```

Comment:
I think store was more appropriate as, it is saving in file and it is persistance. What do you say?

Reply:
The file contains a copy of the Dubbo data in zk. Generally speaking, this copy is also a kind of cache,so I change the "store" to "cache"

Reply:
Hi, @lixiaojiee @khanimteyaz 
Seems like both of u are good.
The difference is not very large. I will deal with it when we reach an agreement.

Reply:
yes..I am fine with this.

Reply:
thanks